### PR TITLE
fix: allow moving columns via keyboard shortcuts

### PR DESCRIPTION
**Pull request by stormojm**

### CODE REVIEW

File: src/features/move-columns/js/column-movable.js [hunk 1]
```diff
@@ -215,6 +215,7 @@
         $timeout(function () {
           grid.api.core.notifyDataChange( uiGridConstants.dataChange.COLUMN );
           grid.api.colMovable.raise.columnPositionChanged(originalColumn.colDef, originalPosition, newPosition);
+          angular.element(`.ui-grid-header-cell.ui-grid-col${originalColumn.uid} .ui-grid-cell-contents`).focus();
```
Comment (JeffRiggle) [inline src/features/move-columns/js/column-movable.js]: I am not sure what browsers this fork needs to support but I do not think IE11 supports string templating

File: src/js/core/directives/ui-grid-header-cell.js [hunk 2]
```diff
@@ -347,6 +347,24 @@
                 }).catch(angular.noop);
             }
 
+            function applyMoveColumn(moveRight) {
+              if (!$scope.col) { return; }
+              if (!uiGridCtrl.grid.api.colMovable) { return; }
```
Comment (JeffRiggle) [inline src/js/core/directives/ui-grid-header-cell.js]: any reason not to combine these two early returns?

Comment (stormojm) [reply src/js/core/directives/ui-grid-header-cell.js]: It was just for readability.  I like this better than it anded.

File: src/features/move-columns/js/column-movable.js (hunk 1 above)
Comment (stormojm) [reply src/features/move-columns/js/column-movable.js]: We already run this through babel so it doesn't matter.  I can switch it though just to keep the set of es6 features small.

Comment (JeffRiggle) [reply src/features/move-columns/js/column-movable.js]: Sure we do but this fork does not use babel. Maybe we just need to update this fork at some point to use babel or update its readme to not include support for IE. In either case that only applies to if others want to use this fork which with 2,000+ I am sure people are more likely to choose something else.

**Pull request merged**